UI/wwwroot/js/chat.js

```diff
@@ -1,17 +1,25 @@
 "use strict";
 
-var connection = new signalR.HubConnectionBuilder().withUrl("/NotificationHub").build();
+var connection = new signalR.HubConnectionBuilder().withUrl("https://localhost:53629/NotificationHub").build();
```

Comment:
pay attention at js code [here](https://learn.microsoft.com/en-us/aspnet/core/tutorials/signalr?view=aspnetcore-8.0&WT.mc_id=dotnet-35129-website&tabs=visual-studio#add-signalr-client-code). I don't think so that we need a full path here

Reply:
For client to client communication we do not require full path but for server to client we required full path else how would connection builder will interact with hub

Reply:
this one needs a bit more time, left as is for now